src/irr/asset/CBAWMeshFileLoader.cpp

```diff
@@ -192,6 +194,147 @@ SAssetBundle CBAWMeshFileLoader::loadAsset(io::IReadFile* _file, const asset::IA
             insertAssetIntoCache(ctx, _override, retval, blobType, hierLvl, thisCacheKey);
 	}
 
+	// flip meshes if needed
+	const core::matrix3x4SIMD xFlip(
+		-1.f,0.f,0.f,0.f,
+		 0.f,1.f,0.f,0.f,
+		 0.f,0.f,1.f,0.f
+	);
+	while (!params.meshesToFlip.empty())
+	{
+		auto mesh = params.meshesToFlip.top();
+		params.meshesToFlip.pop();
+
```

Comment:
Little Optimization: you can skip processing (flipping) the mesh if the reference count `<2` (the stack was the only reference, mesh will get dropped as soon as stack lets go)